registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java

```diff
@@ -468,6 +468,8 @@
 	
 	OSI_VALIDATION_AUTH_SYSTEM_EXCEPTION(PlatformConstants.RPR_OSI_VALIDATOR_MODULE + "008", "Request could not be processed. Please try again"),
 
+	LINK_FOR_USERID_VID_FAILED_OSI_EXCEPTION(PlatformConstants.RPR_OSI_VALIDATOR_MODULE + "009", "Unable to get the Vid for UserId"),
```

Comment:
Here also lets change Vid to individual Id

Reply:
Changed to Vid to individual Id